specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchindex.json

```diff
@@ -245,6 +251,93 @@
               "name": "SearchOptions"
             }
           },
+          {
+            "name": "queryLanguage",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "none",
+              "en-us"
+            ],
+            "x-ms-enum": {
+              "name": "QueryLanguage",
+              "modelAsString": false,
```

Comment:
I think using ModelAsString will allow the customer to enter their own values (which I think is what Luis wanted). We support more than just en-us (for example, we support en)

Reply:
good catch